examples/bq-remote-function/string_formatter/src/main/java/com/google/StringFormat.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google;
```

Comment:
I would consider using a different package name rather than com.google for your custom function specific code and classes. Mainly to differentiate your custom package from that of the actual one of the Google Cloud SDK.

Consider naming it something related to your package example `com.google.cloud.pso.bqremotefunc`

Reply:
thanks, updated the package name